README.md

```diff
@@ -592,3 +593,103 @@ Looking for a specific version of QM?
 Search in the mirrors list below.
 
 [CentOS Automotive SIG - qm package - noarch](https://mirror.stream.centos.org/SIGs/9-stream/automotive/aarch64/packages-main/Packages/q/)
+
+## Changing QM Configuration
+
+### Overview
+
+To run QM on an immutable OSTree-based OS, we utilize systemd units with Podman Quadlet. Since modifying the original service file is not an option, we must use drop-in files to adjust the default configuration.
```

Comment:
Please consider that 

> On an immutable OSTree-based OS, filesystem permissions prevent modifications to the original service file since it is read-only. To configure QM, systemd units with Podman Quadlet can be used, along with drop-in files to adjust the default settings while maintaining system integrity.